README.md

```diff
@@ -17,25 +17,20 @@ You need a working installation of [NodeJS](http://nodejs.org/) and
 [zombie.js](http://zombie.labnotes.org) library through npm:
 
 ``` bash
-npm install -g zombie
+$ npm install -g zombie
 ```
 
 The driver requires zombie.js __version 2.0.0 or higher__.
 
 Use [Composer](https://getcomposer.org/) to install all required PHP dependencies:
 
-```json
-{
-    "require": {
-        "behat/mink":               "~1.6",
-        "behat/mink-zombie-driver": "~1.2"
-    }
-}
+```bash
+$ curl -sS https://getcomposer.org/installer | php
```

Comment:
I would not document how to install Composer here. People not having it already should just use the link above to read the Composer documentation itself instead (yes, I know that this was already there previously).